lib/react-server-route.jsx

```diff
@@ -16,14 +14,14 @@ module.exports = function(req, res, next) {
   // This is essentially a callback lookup. If the requested URL is a known
   // URL based on the routing map as defined in routes.jsx, then this will
   // lead to render properties that can be used to generate page components.
-  ReactRouter.match({ routes, location }, (error, redirectLocation, renderProps) => {
```

Comment:
I used `ReactRouter` for readability- a random `match` method is not obvious behavior